.gitlab/repository.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "vagrant-desktop-x11",
+  "url": "https://gitlab.com/xplatformer/vagrant-desktop-x11",
```

Fix path to gitlab group:

```suggestion
  "url": "https://gitlab.com/xplatformer/env/vagrant-desktop-x11",
```